src/components/tab/Tab.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { STab } from "./styles/STab";
+
+type ITabSize = "Micro" | "Small" | "Medium" | "Default";
+
+interface ITab {
+  size?: ITabSize;
+  activated?: boolean;
+  icon?: JSX.Element;
```

Comment:
There is missing an event handler to set the activated state.

Reply:
I didn't handle it internally since the value of activated can change from outside the component, should I do something like this?
<img width="555" alt="Captura de Pantalla 2021-11-29 a la(s) 16 12 44" src="https://user-images.githubusercontent.com/69903081/143928392-57122bfb-970d-4d2e-b8a7-b39995eda1d0.png">

Reply:
@nicolasshiken but if this tab is part of a series of tab, the selection won't be exclusive between them. Or I'm missing something?

Reply:
I thought that only one Tab could be active at a time, maybe @MoritzGruber can correct me

Reply:
Correct, so if you have 3 tabs [tab 1] [tab 2 *] [tab 3].

Reply:
Oh I think I understand now

Reply:
`There is missing an event handler to set the activated state.` 
I don't understand what you mean with that then

Reply:
If activated state is being passed as props, there should be a event handler passed as well to tigger the activation, like onClick or something.